src/electron/office/document_client.cc

```diff
@@ -359,6 +360,27 @@ void DocumentClient::Emit(const std::string& event_name,
 
 DocumentClient::DocumentClient() = default;
 
+v8::Local<v8::Value> DocumentClient::GotoOutline(int idx, gin::Arguments* args) {
+  char* result = document_->gotoOutline(idx);
+  v8::Isolate* isolate = args->isolate();
+
+  v8::MaybeLocal<v8::String> maybe_res_json_str =
+      v8::String::NewFromUtf8(isolate, result);
```

Comment:
Missing the case where result is `nullptr`, shouldn't happen, but can because C is fun and LOK is fragile.

See here for an example:
https://github.com/coparse-inc/electron-libreoffice/blob/df3f33ca61b9784cd22426ada5125528ffa6fd72/src/electron/office/document_client.cc#L576-L582